extras/cronwrapper

```diff
@@ -48,10 +48,10 @@ LOGFILE=$(mktemp /tmp/plexupdate.cron.XXXX)
 
 RET=0
 if $LOGGING; then
-	"${SCRIPT}" "${CONF}" 2>&1 | tee ${LOGFILE} | logger -t plexupdate -p daemon.info
+	"${SCRIPT}" --config "${CONF}" --notify-success 2>&1 | tee ${LOGFILE} | logger -t plexupdate -p daemon.info
```

Comment:
Must be cross-eyed, didn't see the config :) and yes, I agree with the notify part.